datahub-frontend/app/auth/sso/oidc/custom/CustomOidcAuthenticator.java

```diff
@@ -132,6 +130,16 @@ private static ClientAuthenticationMethod firstSupportedMethod(final List<Client
     }
   }
 
+  private TokenRequest createTokenRequest(final AuthorizationGrant grant) {
```

Comment:
nice! One small request - we tend to put invoked private methods below public methods as code style. Can you move this method to lower down in the file?